olm-catalog/serverless-operator/project.yaml

```diff
@@ -29,6 +29,9 @@ dependencies:
 
   eventing: 1.1.0
   eventing_kafka: 1.1.0
-  eventing_kafka_broker: 1.1.3
+  # eventing-kafka-broker promotion version without the `knative-v` prefix
+  eventing_kafka_broker: 1.2.3
+  # eventing-kafka-broker midstream branch name
+  eventing_kafka_broker_artifacts_branch: release-v1.2
   cli: 1.1.0
```

Comment:
Add a comment as a guideline to always use the promotion version to have 0 for patch version

Reply:
+1